projects/plugins/jetpack/_inc/client/at-a-glance/backup-upgrade/popup.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import { imagePath } from 'constants/urls';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import React from 'react';
+
+export const Popup = ( { posts, comments, onClose } ) => {
+	return (
+		<div className="jp-dash-upgrade-backup__popup">
+			<Button
+				onClick={ onClose }
+				className="jp-dash-upgrade-backup__popup-close-icon"
+				compact
+				borderless
+			>
+				<img src={ imagePath + '/close.svg' } alt={ __( 'Close', 'jetpack' ) } />
+			</Button>
+			<img
+				className="jp-dash-upgrade-backup__popup-lock-icon"
+				src={ imagePath + '/lock.svg' }
+				alt={ __( 'Locked', 'jetpack' ) }
+			/>
+			<p className="jp-dash-upgrade-backup__popup-title">
+				{ __( 'Upgrade to backup', 'jetpack' ) }
+			</p>
+			<p>
+				{ sprintf(
+					/* translators: 1 number of posts, 2 number of comments */
+					__(
+						'Be sure your %1$d posts, %2$d comments and more information is safely backed up with Jetpack',
+						'jetpack'
+					),
+					posts,
+					comments
+				) }
+				{ '' }
```

Comment:
Is this empty string fragment doing anything? I wonder if it might be safely removed.

Reply:
No, it doesn't and I didn't add it. It was probably added by pre-commit hook that runs prettier. If ESLint/Prettier would have been working it should have been caught while writing the code.